GPflowOpt/acquisition/qei_cl.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 Joachim van der Herten
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from .ei import ExpectedImprovement
+import numpy as np
+
+
+class QEI_CL(ExpectedImprovement):
+    """
+    This class is an implementation of the constant liar heuristic (min case)
+    for using Expected Improvement in the batch case.
+
+    See:
+    Ginsbourger D., Le Riche R., Carraro L. (2010)
+    Kriging Is Well-Suited to Parallelize Optimization.
+    """
+
+    def __init__(self, model, batch_size):
```

Comment:
I was thinking to derive batch_size in acquisition each time set_batch is called, so we can keep batch_size in BayesianOptimizer. This would also permit changing the size of the batches during the optimization.

Reply:
I agree